lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_firebase_test.rb

```diff
@@ -0,0 +1,168 @@
+            type: String,
+            verify_block: proc do |value|
+              UI.user_error!("Invalid key file path: #{value}") unless File.exist?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :apk_path,
+            description: 'Path to the application APK on the local machine',
+            type: String,
+            verify_block: proc do |value|
+              UI.user_error!("Invalid application APK: #{value}") unless File.exist?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :test_apk_path,
+            description: 'Path to the test bundle APK on the local machine',
+            type: String,
+            verify_block: proc do |value|
+              UI.user_error!("Invalid test APK: #{value}") unless File.exist?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :model,
+            description: 'The device model to use to run the test',
+            type: String,
+            verify_block: proc do |value|
+              FirebaseTestRunner.verify_has_gcloud_binary!
+              model_names = Fastlane::FirebaseDevice.valid_model_names
+              UI.user_error!("Invalid Model Name: #{value}. Valid Model Names: #{model_names}") unless model_names.include?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :version,
+            description: 'The Android version (API Level) to use to run the test',
+            type: Integer,
+            verify_block: proc do |value|
+              FirebaseTestRunner.verify_has_gcloud_binary!
+              version_numbers = Fastlane::FirebaseDevice.valid_version_numbers
+              UI.user_error!("Invalid Version Number: #{value}. Valid Version Numbers: #{version_numbers}") unless version_numbers.include?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :locale,
+            description: 'The locale code to use when running the test',
+            type: String,
+            default_value: 'en',
+            verify_block: proc do |value|
+              FirebaseTestRunner.verify_has_gcloud_binary!
+              locale_codes = Fastlane::FirebaseDevice.valid_locales
+              UI.user_error!("Invalid Locale: #{value}. Valid Locales: #{locale_codes}") unless locale_codes.include?(value)
+            end
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :orientation,
+            description: 'Which orientation to run the device in',
+            type: String,
+            default_value: 'portrait',
+            verify_block: proc do |value|
+              orientations = Fastlane::FirebaseDevice.valid_orientations
+              UI.user_error!("Invalid Orientation: #{value}. Valid Orientations: #{orientations}") unless orientations.include?(value)
+            end
+          ),
```

Comment:
I just realized something: this API means that we will only be able to pass a single {device,model,version,locale} tuple, as opposed to a list/matrix of devices. But in a couple of client apps we need CircleCI to run on multiple devices (Typically [here to run the screenshots for WPAndroid](https://github.com/wordpress-mobile/WordPress-Android/blob/cd06b20d401f50e4b215df69c5a002a4e3f6b1ae/.circleci/config.yml#L206-L215)).

Given how long this PR has been waiting I'm still inclined to accept this limitation for the first iteration… but this means that if/when we'll want to add support to trigger a FTL run for multiple devices at once, that will incur a breaking change in the API of this action 😒 

Note: We might be able to make-do with the limitation in the meantime, by having separate Buildkite steps in our client pipeline for each {device,locale} we want to trigger… and then create a dependency between them all. But that won't be as optimal as a single trigger though, as that will mean uploading the 2 APK files to Firebase… as many times as we have devices we want to test on — so waste of CI time, bandwidth and FTL quota. Especially for the case of screenshots, when we want to run them on at least {2 devices}x{16 locales}, which will mean 64 APK uploads and 32 _separate_ Buildkite Steps (!) **and** FTL runs (and seemingly unrelated when looking at the FTL UI, as opposed to being represented as a run matrix). Clearly not sustainable.

So I still think that we should fix this in a future iteration, at least to support the case of screenshots.